apps/juxtaposition-ui/src/services/juxt-web/views/web/icons.tsx

```diff
@@ -0,0 +1,27 @@
+import heartIcon from '../../../../webfiles/web/partials/assets/heart_icon.svg?raw';
+import replyIcon from '../../../../webfiles/web/partials/assets/reply_icon.svg?raw';
+import menuIcon from '../../../../webfiles/web/partials/assets/menu_icon.svg?raw';
+import flagIcon from '../../../../webfiles/web/partials/assets/flag_icon.svg?raw';
+import binIcon from '../../../../webfiles/web/partials/assets/bin_icon.svg?raw';
+import shareIcon from '../../../../webfiles/web/partials/assets/share_icon.svg?raw';
```

Comment:
:crying_cat_face:

Reply:
Unfortunately the esbuild plugin im using doesn't support path aliases. So I'd say we keep it like this until we find a better plugin (or method)